lib/js/dom.nim

```diff
@@ -226,20 +231,23 @@ type
     valid*: bool
     valueMissing*: bool
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/Blob
-  Blob* = ref BlobObj
+  Blob* = ref BlobObj ## `https://developer.mozilla.org/en-US/docs/Web/API/Blob`_
   BlobObj {.importc.} = object of RootObj
     size*: int
     `type`*: cstring
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/File
-  File* = ref FileObj
+  File* = ref FileObj ## `https://developer.mozilla.org/en-US/docs/Web/API/File`_
   FileObj {.importc.} = object of Blob
     lastModified*: int
     name*: cstring
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement
-  InputElement* = ref InputElementObj
+  TextAreaElement* = ref object of ElementObj ## `https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement`_
```

Comment:
question for other reviewers:
I notice other types here use this pattern:
```nim
type
 FormElement* = ref FormObj
  FormObj {.importc.} = object of ElementObj
```
however here it's without the importc:
```nim
TextAreaElement* = ref object of ElementObj
```
and, unless I'm missing something, it seems to work too;

what's the point of `{.importc.}` for types with `nim js` ?

Reply:
Reason: Legacy code.

Reply:
you mean, it was there for historical reasons but isn't needed anymore and can be omitted?
or you mean something else?

Reply:
I don't know why we are doing the Obj thing. But I made it consistent:
```nim
  TextAreaElement* = ref TextAreaElementObj 
  TextAreaElementObj {.importc.} = object of Element
```

Reply:
It was written before ``ref object`` was added to Nim, so yes, now it can be omitted.